mysite/templates/psa/modal.html

```diff
@@ -27,7 +27,7 @@ <h4 class="modal-title">Re-send activation link</h4>
   </div>
 
 <div id="email-modal" class="modal fade">
-  <form action="{% url "social:complete" "email" %}" method="post" role="form">{% csrf_token %}
+  <form action="{% url "social:complete" "email" %}?next={{ next }}" method="post" role="form">{% csrf_token %}
```

Comment:
Just wanted to ask whether mixing GET-style params into a POST this way (rather than passing next as a hidden POST parameter) is considered best-practice.  I'll defer to the experts on this one.